recipes/lammps/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cmake ./cmake -DENABLE_ASPHERE=ON -DENABLE_BODY=ON -DENABLE_CLASS2=ON -DENABLE_COLLOID=ON -DENABLE_COMPRESS=OFF -DENABLE_CORESHELL=ON -DENABLE_DIPOLE=ON -DENABLE_GRANULAR=ON -DENABLE_KSPACE=ON -DENABLE_MANYBODY=ON -DENABLE_MC=ON -DENABLE_MEAM=ON -DENABLE_MISC=ON -DENABLE_MOLECULE=ON -DENABLE_PERI=ON -DENABLE_REAX=ON -DENABLE_REPLICA=ON -DENABLE_RIGID=ON -DENABLE_SHOCK=ON -DENABLE_SNAP=ON -DENABLE_SRD=ON -DENABLE_OPT=ON -DENABLE_KIM=OFF -DENABLE_GPU=OFF -DENABLE_KOKKOS=OFF -DENABLE_MPIIO=OFF -DENABLE_MSCG=OFF -DENABLE_VORONOI=OFF -DENABLE_LATTE=OFF -DENABLE_PYTHON=OFF -DENABLE_USER-MEAMC=ON -DENABLE_USER-PHONON=ON -DENABLE_GZIP=ON
+make 
+cp lmp $PREFIX/bin/lmp_serial
+# MPI version are only compiled for Linux 
+if [ $OSX_ARCH != "i386" ] && [ $OSX_ARCH != "x86_64" ]
+then
+    cmake ./cmake -DENABLE_ASPHERE=ON -DENABLE_BODY=ON -DENABLE_CLASS2=ON -DENABLE_COLLOID=ON -DENABLE_COMPRESS=OFF -DENABLE_CORESHELL=ON -DENABLE_DIPOLE=ON -DENABLE_GRANULAR=ON -DENABLE_KSPACE=ON -DENABLE_MANYBODY=ON -DENABLE_MC=ON -DENABLE_MEAM=ON -DENABLE_MISC=ON -DENABLE_MOLECULE=ON -DENABLE_PERI=ON -DENABLE_REAX=ON -DENABLE_REPLICA=ON -DENABLE_RIGID=ON -DENABLE_SHOCK=ON -DENABLE_SNAP=ON -DENABLE_SRD=ON -DENABLE_OPT=ON -DENABLE_KIM=OFF -DENABLE_GPU=OFF -DENABLE_KOKKOS=OFF -DENABLE_MPIIO=OFF -DENABLE_MSCG=OFF -DENABLE_VORONOI=OFF -DENABLE_LATTE=OFF -DENABLE_PYTHON=OFF -DENABLE_USER-MEAMC=ON -DENABLE_USER-PHONON=ON -DENABLE_GZIP=ON -DENABLE_MPI=ON
```

Comment:
Do you really have to call `cmake` three times? Does it not work to just do
```
cmake ./cmake -DENABLE_ASPHERE=ON [...] -DENABLE_GZIP=ON -DBUILD_SHARED_LIBS=ON -DENABLE_MPI=$( [[ $(uname) == "Linux" ]] && echo ON || echo OFF )

make
cp [...]
cd python; python install.py
```

It might not, I don't know, I haven't really looked into it.

But if you do have to call it several times, can we avoid copy-pasting this huge line of arguments? If you want to change one in the future, it'll be easier if you do something like
```
args="-DENABLE_ASPHERE=ON [...] -DENABLE_GZIP=ON"
cmake ./cmake $args
[...]
cmake ./cmake $args -DENABLE_MPI=ON
```